tests/lib/ocpp/v201/test_smart_charging_handler.cpp

```diff
@@ -577,4 +577,49 @@ TEST_F(ChargepointTestFixtureV201,
     EXPECT_THAT(handler.get_profiles(), testing::Contains(profile));
 }
 
+TEST_F(ChargepointTestFixtureV201,
+       K01FR05_IfProfileWithSameIdExistsAndIsNotChargingStationExternalContraints_ThenProfileIsReplaced) {
```

Comment:
Really feel like these test names are all over the place. The original goal was to make them scannable and not to use words that can be easily assumed. 

Words like If the Then are not needed since the location tells us that they're a Given a when or a then. In this name you're missing the Given. Something like:

`AddProfile_ProfileWithSameIdExistsAndNotChargingStationExternalContraints_ProfileReplaced`. Since the names are so long, every letter you can get rid of makes them easier to scan and understand.

Reply:
I personally do not agree with the changes to the wording. The current pattern we have is very easy for me to read and process as a sentence. In any case, if this should be changed, I think that change should be in a separate PR where we update *all* of the test names to follow any given new convention.

Reply:
This isn't a change. There's been a divergence between how we've been naming it. If we want to change how I set it up initially, we should discuss this. 

The rational behind it isn't how easy it is for you to read it, but instead for casual scanners. I started doing this a long time ago because I get migraines.